cmd/push.go

```diff
@@ -26,35 +29,75 @@ var Push = func(providers app.Providers) *cli.Command {
 		},
 		Action: func(c *cli.Context) error {
 			return AppWorkspaceAction(c, func(ctx context.Context, cfg api.Config, appDetails app.Details) error {
-				source, version := c.String("source"), DetectAppVersion(c)
-				osWriters := logging.StandardOsWriters{}
```

Comment:
I would keep the `osWriters` and pass them through instead of directly referencing `os.Stderr`.

Reply:
ok, updated to initialize the osWriters up front and then pass through to the push function.